### PR TITLE
fix(format): Force parents & siblings to have string IDs

### LINKED ISSUE #195: Support for string module & chunk ids
I'm trying to load up some stats files created using the `webpack.NamedChunksPlugin` by creating my own hash.

This breaks with the following error: 
```
Expected number, got "fdef"
  in array at index 0
  at field 'chunks'
  in array at index 1
  at field 'assets'
```

How can I get this working?

### PR DESCRIPTION
**Pull request by Whoaa512**
Follow up from #205 to fix #195

### CODE REVIEW

**Comment by erykpiast** [comment]
Thank you a lot for this! I hope everything will work for you from now on :)

File: playground/named/c/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const crypto = require('crypto');
+const { NamedChunksPlugin } = require('webpack');
+
+module.exports = {
+  entry: './src/index.js',
+  plugins: [
+    new NamedChunksPlugin((chunk) => chunk.name || crypto
+      .createHash('sha256')
+      .update(`${chunk.id}_${Math.random()}`)
```
Comment (erykpiast) [inline playground/named/c/webpack.config.js]: @Whoaa512 The question unrelated to the PR itself: do you use such a pattern in your codebase? I mean exactly this one, naming a chunk with a random value? If the answer is "yes", what for do you need that?

Comment (Whoaa512) [reply playground/named/c/webpack.config.js]: 😅 No I just wanted the chunks to have different names. I tried just using `chunk.id` but the ids are numbers so it didn't have enough entropy to be unique

Comment (erykpiast) [reply playground/named/c/webpack.config.js]: Alright, makes sense! :D

**Pull request merged**